Weblate: Update translations

photoprismci opened the pull request.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/photoprism/photoprism?pullRequest=2130) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/photoprism/photoprism?pullRequest=2130) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/photoprism/photoprism?pullRequest=2130) it.</sub>

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translation: PhotoPrism/Backend
Translate-URL: https://translate.photoprism.app/projects/photoprism/backend/

weblate committed:
Translated using Weblate (Slovak)

Currently translated at 100.0% (74 of 74 strings)

Translation: PhotoPrism/Backend
Translate-URL: https://translate.photoprism.app/projects/photoprism/backend/sk/

weblate committed:
Translated using Weblate (French)

Currently translated at 100.0% (74 of 74 strings)

Translation: PhotoPrism/Backend
Translate-URL: https://translate.photoprism.app/projects/photoprism/backend/fr/

weblate committed:
Translated using Weblate (Romanian)

Currently translated at 100.0% (74 of 74 strings)

Translation: PhotoPrism/Backend
Translate-URL: https://translate.photoprism.app/projects/photoprism/backend/ro/

Translated using Weblate (Romanian)

Currently translated at 100.0% (74 of 74 strings)

Translation: PhotoPrism/Backend
Translate-URL: https://translate.photoprism.app/projects/photoprism/backend/ro/

weblate committed:
Translated using Weblate (Japanese)

Currently translated at 100.0% (73 of 73 strings)

Translation: PhotoPrism/Backend
Translate-URL: https://translate.photoprism.app/projects/photoprism/backend/ja/
